.travis.yml

```diff
@@ -20,14 +20,16 @@ env:
       - OS=fedora DIST=26
       - OS=fedora DIST=27
 
-install:
-  - sudo apt-get install coreutils
-
 script:
+  # lulpeg tags have the following format: major.minor
```

Comment:
It is not so:

PAGER=cat git tag --list
v0.1

Reply:
Why?
0 -> major
1 -> minor
Or you talk about 'v'?

Reply:
Yep, about 'v'.

Reply:
Comment is fixed now.